### ISSUE TITLE
Testrun does not add the programm to the top of the deck

### ISSUE THREAD

**Issue by Rainb00m** [mtgred/netrunner#762]
Problem: Testrun does not add the programm to the top of the deck, even if it is still installed at the end of the turn.

Keep up the good work! You are awesome! =)

**Comment by nealterrell** [comment]
I can't reproduce this. Could you let me know what the circumstances were? Did you by chance move the card onto a host (like Dinosaurus or Scheherezade) after Test Running it?

**Comment by Rainb00m** [comment]
Nope, but at the time the end of the turn I was under effect of "Lockdown" and I used "professional contacts" the next turn to draw the card expecting a "sage" but instead I got a "Silencer". We had some lagging so I clicked two times on "professional contacts". I searched the protocol if the con-player had done something that effects my stack, but nothing. Oh and i had -1 scored Agenda at this time. Beside that nothing unusual. But if you cant reproduce it, delete this comment. If it happens again I will copy the protocol. =)
